src/library_pthread.js

```diff
@@ -660,6 +660,18 @@ var LibraryPThread = {
     return navigator['hardwareConcurrency'];
   },
 
+  emscripten_thread_pool_size: function() {
+#if PTHREAD_POOL_SIZE
```

Comment:
I think the entire body could be `return {{{ PTHREAD_POOL_SIZE || 0 }}}`.

(`{{{ }}}` evaluates at compile time, and the default value of that setting is the empty string)